src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -1345,7 +1345,7 @@ async Task UseAsync()
         result = await lambda();
     }
 }";
-            await TestAsync(markup, MainDescription($"({CSharpFeaturesResources.awaitable}) {string.Format(FeaturesResources.Awaited_task_returns_0, "class System.Threading.Tasks.Task<TResult>")}"),
+            await TestAsync(markup, MainDescription(string.Format(FeaturesResources.Awaited_task_returns_0, $"({CSharpFeaturesResources.awaitable}) class System.Threading.Tasks.Task<TResult>")),
```

Comment:
This is technically a behavior change that I don't care enough about to fix. In the case of a nested Task previously Quick Info would show

> (awaitable) Awaited task returns 'Task<string>'

and now it shows

> Awaited task returns '(awaitable) Task<string>'

Arguably it makes it a bit clearer that the awaitable thing being tagged is the return type, instead of some potential confusion of whether the await is implicitly why an awaitable is being shown.

Reply:
It's potentially less clear if you are somebody who really wants that (awaitable) in the front. I'm going to gather that if you're somebody using tasks returning tasks, you are probably needing the hint at all. ;-)